Program/Program.cs

```diff
@@ -7,17 +8,46 @@ namespace Program
 {
     class Program
     {
+        private static HotelManager _manager;
+        private static ILogger _logger;
+
         static void Main(string[] args)
```

Comment:
Should have:

Program.cs file/class is stricly related to one type of project console application. It is always worth to separate your logic from proram files. In your case I would suggest to move things like:
* Setup Objects
* Valdiation

Generally program file should as samll as possible